docs/inputs.md

```diff
@@ -441,6 +450,7 @@ An array of strings (or objects with a `text` property, if you add the `property
 * **badge** - name of the icon (or a two-character string) that should be displayed in the simple list item when editing. Icon names can be anything from the [Material Design Icon Set](https://material.io/icons/), or you can use two initials
 * **allowRepeatedItems** - allow the same item more than once. defaults to false
 * **autocomplete** - object with autocomplete options. Currently this is just the key `list` where the value is the name of a list that Amphora knows about accessible via `/<site>/_lists/<listName>`. Example:
+* **modifyList** - allow the user modify the list that is passed into autocomplete. This will send a PUT to Amphora. set `allowDestory` to true, to let the user remove items from the list via the autocomplete UI
```

Comment:
Per above, I wouldn't get into the internal details of `allowDestroy`. Rather, just discuss the exposed configuration option and what it does from a user perspective (enabling a UI affordance to remove items from the autocomplete list)

Also, will new items automatically be _added_ to the list if this is enabled? I think that's what "this will sent a PUT to amphora" means, but it's not super clear.

By the by, you should probably change the line above to reflect the new configuration options allowed.